patches/jest-environment-jsdom+27.5.1.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/node_modules/jest-environment-jsdom/build/index.js b/node_modules/jest-environment-jsdom/build/index.js
+index 77176ba..1644673 100644
+--- a/node_modules/jest-environment-jsdom/build/index.js
++++ b/node_modules/jest-environment-jsdom/build/index.js
+@@ -102,6 +102,7 @@ class JSDOMEnvironment {
+     (0, _jestUtil().installCommonGlobals)(global, config.globals); // TODO: remove this ASAP, but it currently causes tests to run really slow
+ 
+     global.Buffer = Buffer; // Report uncaught errors.
++    global.Uint8Array = Uint8Array;
```

Comment:
How did you find this solution? 🙌

Reply:
I found this line in the node test env: https://github.com/facebook/jest/blob/1b402ab76b8bd0f8c5c8512345e2fd7aca8d83ed/packages/jest-environment-node/src/index.ts#L46

In most issues people advice to define the node test env, which in our case doesn't work.